### PR TITLE
Disable tests, which are not ready to be run yet 

### PR DESCRIPTION
**Pull request by mnoszczak**
# Description

Disable tests, which are not ready to be run yet 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document change (fix typo or modifying any markdown files, code comments or anything in the examples folder only)

## All Submissions

- [ ] Have you followed the guidelines in our Contributing document?
- [ ] Have you provided a description?
- [ ] Are your changes properly formatted?

## New Feature Submissions

- [ ] Does your submission pass tests?
- [ ] Have you added thorough tests for your new feature?
- [ ] Have you commented your code, particularly in hard-to-understand areas?
- [ ] Have you added a Docstring?

## Changes to Core Features

- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully run tests with your changes locally?
- [ ] Have you updated any code comments, as applicable?

### CODE REVIEW

**Review by mrobers1982 — APPROVED** [review]

**Review by tomislav-peharda — APPROVED** [review]

**Comment by vbrodsky** [comment]
Please do not merge this, our process for running sdk integration tests for codefresh has changed and this will not help to unblock it

**Review by adrian-chang — APPROVED** [review]

**Comment by adrian-chang** [comment]
Closing as they are enabled / passing on develop

**Pull request closed without merging**
